src/docfx/lib/schema/JsonSchemaTransformer.cs

```diff
@@ -413,6 +417,8 @@ private string GetXrefHref(FilePath file, string uid, int uidCount, bool isRootL
             switch (schema.ContentType)
             {
                 case JsonSchemaContentType.Href:
+                    ValidateLink(file, new HyperLinkNode { HyperLinkType = HyperLinkType.Default, UrlLink = stringValue, SourceInfo = sourceInfo });
```

Comment:
Try consolidate to use `LinkResolver` as the entry for all link validation

Reply:
good idea, if we can one entry for all link resolve and link validaiton, that would be great

Reply:
Done. Move validatelink into LinkResolver to make it consistent.